pkg/kudoctl/cmd/init.go

```diff
@@ -91,6 +92,7 @@ func newInitCmd(fs afero.Fs, out io.Writer) *cobra.Command {
 	f.BoolVarP(&i.clientOnly, "client-only", "c", false, "If set does not install KUDO on the server")
 	f.StringVarP(&i.image, "kudo-image", "i", "", "Override KUDO controller image and/or version")
 	f.StringVarP(&i.version, "version", "", "", "Override KUDO controller version of the KUDO image")
+	f.StringVarP(&i.ns, "controller-namespace", "", "", "Override namespace for the KUDO controller. Defaults to kudo-system")
```

Comment:
why would we not use the existing and global `namespace` flag?   in the context of `init` `--namespace` would mean to me the namespace to init to .

Reply:
Done, now I just have to find a way to set the default namespace for the `init` command.